packages/compiler/test/cli.test.ts

```diff
@@ -34,6 +34,15 @@ describe("compiler: cli", () => {
       });
     });
 
+    it("--config plumbs through", async () => {
+      const options = await resolveCompilerOptions({ config: `${cwd}/myConfigDir` });
```

Comment:
We should add tests for providing a file path or a directory path

Reply:
We should likely also add tests that emit the new error diagnostic for config not found / inaccessible, if that is possible

Reply:
Yeah, I didn't see any tests of that nature in the existing code

Reply:
Added tests.